### ISSUE TITLE
Build failure: logrus: use of vendored package not allowed (go 1.6)

### ISSUE THREAD

**Issue by npcode** [twistlock/authz#11]
Here is what I did:
```
$ cd authz
$ go get -u
$ cd ..
$ make
golint authz/basic.go || exit; golint authz/basic_test.go || exit; golint authz/doc.go || exit; golint broker/main.go || exit; golint core/doc.go || exit; golint core/interfaces.go || exit; golint core/route_parser.go || exit; golint core/route_parser_test.go || exit; golint core/server.go || exit; golint core/types.go || exit;
gofmt -w authz/basic.go authz/basic_test.go authz/doc.go broker/main.go core/doc.go core/interfaces.go core/route_parser.go core/route_parser_test.go core/server.go core/types.go
go vet ./core/.; go vet ./broker/.; go vet ./authz/.;
CGO_ENABLED=0 go build  -o authz-broker -a -installsuffix cgo ./broker/main.go
package main
	imports github.com/docker/docker/vendor/src/github.com/Sirupsen/logrus: use of vendored package not allowed
Makefile:26: recipe for target 'binary' failed
make: *** [binary] Error 1
```

My go version is 1.6:
```
$ go version
go version go1.6 linux/amd64
```

**Comment by npcode** [comment]
It was built successfully with go 1.5.

**Comment by npcode** [comment]
It seems to be fixed at 10b1aca6ddfee554f6e86d484d9a13b12359b252. Thanks!
